.stylelintrc.json

```diff
@@ -0,0 +1,32 @@
+{
+    "extends": ["stylelint-config-standard"],
+    "plugins": ["stylelint-scss", "stylelint-csstree-validator"],
+    "rules": {
+      "at-rule-no-unknown": [
+        true,
+        {
+          "ignoreAtRules": [
+            "tailwind",
+            "apply",
+            "variants",
+            "responsive",
+            "screen"
+          ]
+        }
+      ],
+      "scss/at-rule-no-unknown": [
+        true,
+        {
+          "ignoreAtRules": [
+            "tailwind",
+            "apply",
+            "variants",
+            "responsive",
+            "screen"
+          ]
+        }
+      ],
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/235439980-9e99367e-718c-4fe9-838c-82a05c19f44d.png)

**[OPTIONAL]**

- [ ] It would be nice if you make your commit messages more meaningful and capitalized 👍 because some of your commit messages are not capitalized or meaningful enough 😨 Please consider this issue to make your pull request professional and descriptive ☺️